pkg/cmd/server/api/types.go

```diff
@@ -210,6 +210,10 @@ type MasterConfig struct {
 	// EtcdClientInfo contains information about how to connect to etcd
 	EtcdClientInfo EtcdConnectionInfo
 
+	// KubernetesEtcdClientInfo contains information about how to connect to another etcd
+	// for kuberenetes data storage
+	KubernetesEtcdClientInfo *EtcdConnectionInfo
```

Comment:
this should be under KubernetesMasterConfig, it isn't relevant if we aren't running k8s in-process